addons/web/static/src/views/fields/properties/property_definition.xml

```diff
@@ -143,15 +152,18 @@
                     || (state.propertyDefinition.comodel &amp;&amp; state.propertyDefinition.comodel.length))"
                     class="o_field_property_definition_value">
                     <td class="o_td_label align-top pe-2">
-                        <b t-if="state.propertyDefinition.type === 'boolean'">
-                            Default State
-                        </b>
-                        <b t-else="">
-                            Default Value
-                        </b>
+                        <label t-att-for="domInputIdPrefix + '-default'">
```

Comment:
Would do "_default" and not with a "-" for consistency.

Reply:
I made a method to build the id, so it should be consistent everywhere now